tests/utf8/Utf8IntrospectTest.php

```diff
@@ -0,0 +1,140 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+use namespace HH\Lib\Experimental\Str\Utf8;
+use function Facebook\FBExpect\expect;
+// @oss-disable: use InvariantViolationException as InvariantException;
+
+final class Utf8IntrospectTest extends PHPUnit_Framework_TestCase {
+
+  public static function provideLength(): varray<mixed> {
```

Comment:
`varray<(string, int)>` - though you should be able to use `vec<(string ,int)>` with HackTest